charts/trino/values.yaml

```diff
@@ -24,11 +24,14 @@ server:
     path: /etc/trino
     http:
       port: 8080
+      allowInsecureOverHttp: false
```

Comment:
This can be added in `server.coordinatorExtraConfig` or `additionalConfigProperties`. We should not encourage using this, as it reduces the security, so I'm not sure if we want a dedicated value for this. If you disagree, can you open a separate PR with this change?